Modules/Messages/src/Messages.Domain/Entities/Message.cs

```diff
@@ -43,4 +43,19 @@ public void LoadBody(byte[] bytes)
         }
         Body = bytes;
     }
+
+    public void ReplaceIdentityAddress(IdentityAddress oldIdentityAddress, IdentityAddress newIdentityAddress)
+    {
+        if (CreatedBy == oldIdentityAddress)
```

Comment:
Should we also check if `oldIdentityAddress != newIdentityAddress`?

Reply:
In fact, the method should be called `DeleteParticipant`, and it should just accept the address of the Identity that should be deleted. Internally, the method uses our placeholder address to replace all occurrences.

Reply:
I created this method with two parameters because I was not comfortable having a Domain method access the `applicationOptions`, needed for getting the `AddressPrefix` configuration. Does this not make sense?

Reply:
Hmm, yeah... Interesting. Okay, leave it as it is for now. I will have to rethink this whole address prefix thing soon anyway.

And I think you can ignore Daniel's comment ("Should we also check if oldIdentityAddress != newIdentityAddress?) as well. That's not really a problem, I think.